Flowchart/Views/Home/Contact.cshtml

```diff
@@ -1,15 +1,21 @@
 ﻿@{
     ViewData["Title"] = "Contact";
 }
+<head>
+    <style>
+
+    </style>
+</head>
 <h2>@ViewData["Title"].</h2>
+<p>feel free to contact us for assistance</p>
 <dl>
-   <dt>email Sam Messina:</dt>
+   <dt><font size="5">email Sam Messina:</font></dt>
```

Comment:
Move the style for the font to either the dl tag or inside the style header. Probably not the most efficient way to add a font size to text here. [Learn CSS](https://www.w3schools.com/css/)